src/Vester.sol

```diff
@@ -124,7 +125,6 @@ contract Vester is Initializable, IVester {
         }
         vestingPosition.claimed = true;
         // Claim AURA rewards
-        // TODO: Q: should send all AURA rewards even if there are multiple vesting positions?
         AURA_REWARD_POOL.getReward();
```

Comment:
Since we're claiming AURA here, we should emit `ClaimedAuraRewards`. Also, consider adding the AURA claiming logic to an internal function since it is currently duplicated both here and in `claimAuraRewards()`.

Reply:
Done